src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int UPPER_LIMIT = 101;
+    private static Random random = new Random();
+
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(UPPER_LIMIT);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
same lets make ColorSupplier as class level variable, not to create new instances every time we call getRandomBall() method